docs/api.rst

```diff
@@ -8581,24 +8581,33 @@ Getting Loopring balances
 Getting Eth2 Staking details
 ==============================
 
-.. http:get:: /api/(version)/blockchains/ETH2/stake/details
+.. http:post:: /api/(version)/blockchains/ETH2/stake/details
 
-   Doing a GET on the ETH2 stake details endpoint will return detailed information about your ETH2 staking activity.
+   Doing a POST on the ETH2 stake details endpoint will return detailed information about your ETH2 staking activity and information about the earned ETH as part of withdrawals, mev rewards and new blocks.
 
    .. note::
       This endpoint is only available for premium users
 
    .. note::
       This endpoint can also be queried asynchronously by using ``"async_query": true``
 
+
    **Example Request**:
 
    .. http:example:: curl wget httpie python-requests
 
-      GET /api/1/blockchains/ETH2/stake/details HTTP/1.1
+      POST /api/1/blockchains/ETH2/stake/details HTTP/1.1
       Host: localhost:5042
 
-   :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not
+      {'validator_indices': [1111]}
+
+      :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not.
+      :reqjson bool only_cache: If true then only the stats are calculated and the details are empty.
+      :reqjson int from_timestamp: The timestamp from which to start querying. Default is 0.
+      :reqjson int to_timestamp: The timestamp until which to query. Default is now.
+      :reqjson list[string] location_labels: A list of location labels to optionally filter by. Location label is a string field that allows to provide more information about the location. When we use this structure in blockchains, it is used to specify the user address. For exchange events it's the exchange name assigned by the user.
+      :reqjson list[int] validator_indices: An optional list of validator indices to filter by. This makes it an EthStakingevent query
```

Comment:
Do not copy paste from other docs. Give more specific information about how location label is treated **here**

```suggestion
      :reqjson list[int] validator_indices: An optional list of validator indices to filter by.
```